src/vs/workbench/contrib/extensions/browser/extensionsViews.ts

```diff
@@ -708,10 +708,10 @@ export class ExtensionsListView extends ViewletPanel {
 				if (count === 0 && this.isBodyVisible()) {
 					if (error) {
 						if (error instanceof ExtensionListViewWarning) {
-							this.bodyTemplate.messageSeverityIcon.className = SeverityIcon.className(Severity.Warning);
+							this.bodyTemplate.messageSeverityIcon.className = 'codicon ' + SeverityIcon.className(Severity.Warning);
```

Comment:
Can be written more elegant as \`codicon ${SeverityIcon.className(Severity.Warning)}\`